test/Microsoft.DotNet.Tools.Tests.Utilities/RepoDirectoriesProvider.cs

```diff
@@ -58,46 +57,35 @@ public static string RepoRoot
         }
 
         public string Artifacts => _artifacts;
-        public string BuiltDotnet => _builtDotnet;
         public string DotnetRoot => _dotnetRoot;
         public string NugetPackages => _nugetPackages;
         public string Stage2Sdk => _stage2Sdk;
         public string TestPackages => _testPackages;
         public string TestWorkingFolder => _testWorkingFolder;
         public string TestArtifactsFolder => _testArtifactsFolder;
 
-        public RepoDirectoriesProvider(
-            string artifacts = null,
-            string builtDotnet = null,
-            string nugetPackages = null,
-            string corehostPackages = null,
-            string corehostDummyPackages = null)
+        public RepoDirectoriesProvider()
         {
-            //  Ideally this wouldn't be hardcoded, so that you could use stage n to build stage n + 1, and then use stage n + 1 to run tests
-            int previousStage = 2;
+            var configuration =
+                Environment.GetEnvironmentVariable("CONFIGURATION") ?? "Debug";
 
-            _artifacts = artifacts ?? Path.Combine(RepoRoot,
-                                                   "bin",
-                                                   previousStage.ToString());
-            _builtDotnet = builtDotnet ?? Path.Combine(_artifacts, "intermediate", "sharedFrameworkPublish");
-            _dotnetRoot = Path.Combine(_artifacts, "dotnet");
-            _nugetPackages = nugetPackages ?? Path.Combine(RepoRoot, ".nuget", "packages");
+            _artifacts = Path.Combine(RepoRoot, "artifacts");
+            _dotnetRoot = Path.Combine(_artifacts, "tmp", configuration, "dotnet");
+            _nugetPackages = Path.Combine(RepoRoot, ".nuget", "packages");
             _stage2Sdk = Directory
-                .EnumerateDirectories(Path.Combine(_artifacts, "dotnet", "sdk"))
+                .EnumerateDirectories(Path.Combine(_dotnetRoot, "sdk"))
                 .First(d => !d.Contains("NuGetFallbackFolder"));
 
-            _testPackages = Environment.GetEnvironmentVariable("TEST_PACKAGES");
-            if (string.IsNullOrEmpty(_testPackages))
-            {
-                throw new InvalidOperationException("TEST_PACKAGES environment variable not set");
-            }
+            _testPackages = Path.Combine(_artifacts, "tmp", configuration, "test", "packages");
+            // _testPackages = Environment.GetEnvironmentVariable("TEST_PACKAGES");
+            // if (string.IsNullOrEmpty(_testPackages))
+            // {
+            //     throw new InvalidOperationException("TEST_PACKAGES environment variable not set");
+            // }
```

Comment:
Remove commented out code.

Reply:
This is fixed in a later commit. I'm giving up on reviewing commit by commit. It's overwhelming GitHub and CodeFlow.